terasoluna-gfw-functionaltest-parent/pom.xml

```diff
@@ -135,12 +140,13 @@
         <java-version>1.6</java-version>
         <postgresql.version>9.4-1205-jdbc41</postgresql.version>
         <ojdbc7.version>12.1.0.2</ojdbc7.version>
+        <db2jcc4.version>4.19.26</db2jcc4.version>
         <org.codehaus.cargo.cargo-maven2-plugin.version>1.4.16</org.codehaus.cargo.cargo-maven2-plugin.version>
         <cargo.jvmargs>-Xms512m -Xmx1024m -Duser.language=ja -Duser.country=JP -Duser.timezone=Asia/Tokyo</cargo.jvmargs>
         <cargo.daemon.url>http://localhost:18000</cargo.daemon.url>
         <cargo.container.id>tomcat8x</cargo.container.id>
         <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-8/v${tomcat8.version}/bin/apache-tomcat-${tomcat8.version}.zip</cargo.container.url>
-        <tomcat8.version>8.0.29</tomcat8.version>
+        <tomcat8.version>8.0.30</tomcat8.version>

```

Comment:
@ikeyat 
 Sorry, It's difference that may occur due to I had to rebase too much.
 I Canceled the wrong rebase, and re-push only the necessary commits.
 This difference does not occur.